services/audio/.gitignore

```diff
@@ -0,0 +1,38 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+/logs
+*.class
```

Comment:
Not certain this makes a lot of sense, but it'll do for now.  I'd like to know what generates these files.